File: basics/Q38/main.go
package main

import "fmt"

func main() {
	var year float64
	fmt.Print("Enter year: ")
	fmt.Scan(&year)
	day := 365.25 * year
	fmt.Println("Day is", day)
	month := int(day / 30)
	fmt.Println("Month is", month)
	seconds := day * 24 * 3600
	fmt.Println("Seconds is", seconds)
}

File: basics/Q38/main.rs
use std::io;

fn main() {
    let mut input = String::new();

    println!("Enter year:");
    io::stdin().read_line(&mut input).expect("Failed to read line");
    let year: f64 = input.trim().parse().expect("Not a valid number");

    let day: f64 = 365.25 * year;
    println!("Day is {}", day);

    let month: i32 = (day / 30.0) as i32;
    println!("Month is {}", month);

    let seconds: f64 = day * 24.0 * 3600.0;
    println!("Seconds is {}", seconds);
}